1-mandatory/2-quotegenerator/quotes.js

```diff
@@ -1,3 +1,12 @@
+const press = document.getElementById("press");
+const quote = document.getElementById("quote");
+
+press.addEventListener("click", () => {
+  const randomQuote = pickFromArray(quotes);
+  quote.innerHTML = `${randomQuote.quote} <br> Author: <em>${randomQuote.author}</em>`;
+});
+
+
```

Comment:
Really nice Mo!!! 🎉  🕺  🙌 

Nothing else to say here, perfect coding!!! Well done!!!